app/assets/javascripts/admin/addon/components/admin-user-field-item.js

```diff
@@ -75,7 +77,7 @@ export default class AdminUserFieldItem extends Component {
   setRequirement(value, { set }) {
     set("requirement", value);
 
-    if (value === "for_all_users") {
+    if (this._requiredForExistingUsers(value)) {
```

Comment:
I considered adding the relevant predicates to the server-side serializer, but FormKit doesn't operate on the actual model, so it wouldn't help.